### PR TITLE
Fix comments arond class/struct/enum decalrations

### PR DESCRIPTION
**Pull request by 9rnsr**
Get back readability.

### CODE REVIEW

File: src/declaration.d [hunk 1]
```diff
@@ -102,99 +102,74 @@ extern (C++) void ObjectNotFound(Identifier id)
...
-enum STCalias = 0x10000000L;
-// alias parameter
-enum STCshared = 0x20000000L;
-// accessible from multiple threads
-// accessible from multiple threads
-// but not typed as "shared"
-enum STCgshared = 0x40000000L;
-enum STCwild = 0x80000000L;
-// for "wild" type constructor
+enum STCundefined           = 0x_____________0L;
```
Comment (yebblies) [inline src/declaration.d]: Eww no.  Just use `(1 << n)`.

Comment (9rnsr) [inline src/declaration.d]: OK, done.

File: src/attrib.d [hunk 2]
```diff
@@ -37,13 +37,13 @@ import ddmd.tokens;
 import ddmd.utf;
 import ddmd.visitor;
 
-/**************************************************************/
+/***********************************************************
```
Comment (yebblies) [inline src/attrib.d]: Can't we just delete these?  They were much more useful in C++ where the function bodies were not inside the class declarations.

Comment (9rnsr) [inline src/attrib.d]: These are for future ddoc-able comments in dmd source code.

Comment (yebblies) [inline src/attrib.d]: Ok fair enough.

File: src/traits.d [hunk 3]
```diff
@@ -120,6 +120,7 @@ extern (C++) static void collectUnitTests(Dsymbols* symbols, AA* uniqueUnitTests
 }
 
 /************************ TraitsExp ************************************/
```
Comment (yebblies) [inline src/traits.d]: This comment doesn't match the declaration...

File: src/aggregate.d [hunk 4]
```diff
@@ -36,9 +36,9 @@ import ddmd.visitor;
 
 enum Sizeok : int
 {
-    SIZEOKnone, // size of aggregate is not computed yet
-    SIZEOKdone, // size of aggregate is set correctly
-    SIZEOKfwd, // error in computing size of aggregate
+    SIZEOKnone,             // size of aggregate is not computed yet
```
Comment (yebblies) [inline src/aggregate.d]: I'm not a fan of these.  I would like to have dfmt automatically do the alignment, but I think manually maintaining them is a waste of time.

Comment (9rnsr) [inline src/aggregate.d]: I hate to read scrumbled code during maintenance. Reading human unfriendly code wastes MY time.

File: src/traits.d (hunk 3 above)
Comment (9rnsr) [inline src/traits.d]: Yes, I know. It's just a separator comment.

File: src/aggregate.d (hunk 4 above)
Comment (yebblies) [inline src/aggregate.d]: Constant whitespace changes waste everybody's time.

Comment (9rnsr) [inline src/aggregate.d]: "Constant"? How frequently this line will be modified? In cdmd age, the `SIZEOK` definition had not been changed from few years past.

**Comment by 9rnsr** [comment]
@yebblies Why you still prefer the human unfriendly format? Actually the magicport conversion result is broken in some where. #5114 is trying to fix it (and this PR already contains that fix).

**Comment by 9rnsr** [comment]
@WalterBright @MartinNowak

**Comment by yebblies** [comment]
I don't prefer the human unfriendly format, but it has the advantage of matching what dfmt outputs.  I'm completely in favor of improving it if we keep it matching dfmt (or improve dfmt to do pretty alignment etc).

**Comment by 9rnsr** [comment]
@yebblies If the dfmt output is **already** enough human friendly, I can accept your argument. But it's not yet. In the current state, we have **no** benefit to keep scrambled code style.

**Comment by mathias-lang-sociomantic** [comment]
Couldn't dfmt just provide an option to put comment *before* the declaration so we don't have to deal with the alignment ?

File: src/aliasthis.d [hunk 5]
```diff
@@ -24,17 +24,17 @@ import ddmd.root.outbuffer;
...
 import ddmd.visitor;
 
-/**************************************************************/
+/***********************************************************
+ * alias ident this;
+ */
 extern (C++) final class AliasThis : Dsymbol
 {
 public:
-    // alias Identifier this;
```
Comment (WalterBright) [inline src/aliasthis.d]: Why remove this comment? I like to have comments like these that relate source syntax with the corresponding data structure.

Comment (9rnsr) [inline src/aliasthis.d]: This is moved up to line 28.

**Comment by 9rnsr** [comment]
@WalterBright Thank you!

**Comment by WalterBright** [comment]
Although I want to use dfmt, if it doesn't line up the line comments it isn't ready to be used.

**Comment by MartinNowak** [comment]
I'm using a proportional font and don't care about any alignment at all, much more human friendly IMO.

**Pull request merged**